src/hooks/useGetConfig.js

```diff
@@ -30,6 +30,8 @@ const useGetConfig = () => {
     favicon,
     platformName: data?.platform_name,
     gtm: data?.gtm,
+    useTPAOnly: data?.use_tpa_only,
```

Comment:
in terms of best-practice, the 'use' prefix is only usable for hooks so please change useTPAOnly to something like isTPAOnly as I mistook is with a hook

Reply:
Good idea. Changed to isTPAOnly to follow convention.